### PR TITLE
r/glue_crawler - s3 sample size

### LINKED ISSUE #18174: aws_glue_crawler removing configuration block causes continual diff
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version
Terraform: v0.14.8
AWS Provider: v3.32.0

### Affected Resource(s)

* aws_glue_crawler

### Terraform Configuration Files

```hcl
provider "aws" {
  region = "us-east-1"
}

data "aws_caller_identity" "current" {}

resource "aws_glue_catalog_database" "test" {
  name = "test-database"
}

resource "aws_glue_crawler" "test" {
  name          = "test-crawler"
  role          = aws_iam_role.glue.arn
  database_name = aws_glue_catalog_database.test.name

  configuration = jsonencode({
    Version = 1
    Grouping = {
      TableGroupingPolicy = "CombineCompatibleSchemas"
    }
  })

  s3_target {
    path = "s3://${aws_s3_bucket.test.bucket}/"
  }
}

resource "aws_s3_bucket" "test" {
  bucket = "test-bucket-${data.aws_caller_identity.current.account_id}"
}

resource "aws_iam_role" "glue" {
  name               = "test-glue-role"
  assume_role_policy = data.aws_iam_policy_document.glue_assume.json
}

data "aws_iam_policy_document" "glue_assume" {
  statement {
    actions = ["sts:AssumeRole"]

    principals {
      type        = "Service"
      identifiers = ["glue.amazonaws.com"]
    }
  }
}
```

### Debug Output

```log
~> tf apply
aws_glue_catalog_database.test: Refreshing state... [id=374302612192:test-database]
aws_iam_role.glue: Refreshing state... [id=test-glue-role]
aws_s3_bucket.test: Refreshing state... [id=test-bucket-374302612192]
aws_glue_crawler.test: Refreshing state... [id=test-crawler]

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # aws_glue_crawler.test will be updated in-place
  ~ resource "aws_glue_crawler" "test" {
      - configuration = jsonencode(
            {
              - Grouping = {
                  - TableGroupingPolicy = "CombineCompatibleSchemas"
                }
              - Version  = 1
            }
        ) -> null
        id            = "test-crawler"
        name          = "test-crawler"
        tags          = {}
        # (4 unchanged attributes hidden)




        # (4 unchanged blocks hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

aws_glue_crawler.test: Modifying... [id=test-crawler]
aws_glue_crawler.test: Modifications complete after 1s [id=test-crawler]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
~> tf apply
aws_glue_catalog_database.test: Refreshing state... [id=374302612192:test-database]
aws_iam_role.glue: Refreshing state... [id=test-glue-role]
aws_s3_bucket.test: Refreshing state... [id=test-bucket-374302612192]
aws_glue_crawler.test: Refreshing state... [id=test-crawler]

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # aws_glue_crawler.test will be updated in-place
  ~ resource "aws_glue_crawler" "test" {
      - configuration = jsonencode(
            {
              - Grouping = {
                  - TableGroupingPolicy = "CombineCompatibleSchemas"
                }
              - Version  = 1
            }
        ) -> null
        id            = "test-crawler"
        name          = "test-crawler"
        tags          = {}
        # (4 unchanged attributes hidden)




        # (4 unchanged blocks hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

aws_glue_crawler.test: Modifying... [id=test-crawler]
aws_glue_crawler.test: Modifications complete after 1s [id=test-crawler]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
~>
```

### Expected Behavior

Terraform should correctly remove the configuration from the Glue crawler.

### Actual Behavior

Terraform has a continual diff if you remove the configuration block.

### Steps to Reproduce

1. `terraform apply`
2. Comment out `configuration` block in resource `aws_glue_crawler`
3. `terraform apply` -> Terraform will report that it will remove the configuration
4. Check the console. You will see that the "combine compatible schemas" option is still enabled.
4. `terraform apply` again and again -> Terraform will *always* report that it will remove the configuration

### Important Factoids

* Nothing special was done; this Terraform was run on an empty AWS account.
* Removing only the `Grouping` block inside the `configuration` block also causes a continual diff.

### References

* #0000

### LINKED ISSUE #20177: aws_glue_crawler : s3_target doesn't support sample size
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description
AWS Glue Crawler API supports specifying sample_size for Amazon S3 data stores (aka s3_targets) https://docs.aws.amazon.com/glue/latest/dg/define-crawler.html

This functionality is not supported in terraform AWS Provider yet. This feature can reduce crawl time significantly. 

### New or Affected Resource(s)

* aws_glue_crawler

### Potential Terraform Configuration

```hcl
resource "aws_glue_crawler" "example" {
  database_name = aws_glue_catalog_database.example.name
  name          = "example"
  role          = aws_iam_role.example.arn

  s3_target {
    path = "s3://${aws_s3_bucket.example.bucket}"
    sample_size = 120
  }
}
```

### References

* #0000

### PR DESCRIPTION
**Pull request by DrFaust92**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Closes #20177
Closes #18174

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccAWSGlueCrawler_'
--- PASS: TestAccAWSGlueCrawler_Role_Name_Path (56.98s)
--- PASS: TestAccAWSGlueCrawler_Role_ARN_Path (59.48s)
--- PASS: TestAccAWSGlueCrawler_disappears (65.08s)
--- PASS: TestAccAWSGlueCrawler_Role_ARN_NoPath (80.63s)
--- PASS: TestAccAWSGlueCrawler_SchemaChangePolicy (131.69s)
--- PASS: TestAccAWSGlueCrawler_DynamodbTarget (148.54s)
--- PASS: TestAccAWSGlueCrawler_mongoDBTarget (148.54s)
--- PASS: TestAccAWSGlueCrawler_Description (149.07s)
--- PASS: TestAccAWSGlueCrawler_S3Target_Exclusions (150.05s)
--- PASS: TestAccAWSGlueCrawler_Configuration (150.29s)
--- PASS: TestAccAWSGlueCrawler_JdbcTarget_Exclusions (153.17s)
--- PASS: TestAccAWSGlueCrawler_SecurityConfiguration (158.23s)
--- PASS: TestAccAWSGlueCrawler_Schedule (197.96s)
--- PASS: TestAccAWSGlueCrawler_RemoveTablePrefix (146.95s)
--- PASS: TestAccAWSGlueCrawler_Classifiers (206.54s)
--- PASS: TestAccAWSGlueCrawler_JdbcTarget_Multiple (207.00s)
--- PASS: TestAccAWSGlueCrawler_recrawlPolicy (208.01s)
--- PASS: TestAccAWSGlueCrawler_Tags (210.34s)
--- PASS: TestAccAWSGlueCrawler_mongoDBTarget_multiple (211.36s)
--- PASS: TestAccAWSGlueCrawler_mongoDBTarget_scan_all (211.42s)
--- PASS: TestAccAWSGlueCrawler_lineageConfig (211.67s)
--- PASS: TestAccAWSGlueCrawler_TablePrefix (149.02s)
--- PASS: TestAccAWSGlueCrawler_S3Target_ConnectionName (90.00s)
--- PASS: TestAccAWSGlueCrawler_DynamodbTarget_scanRate (184.81s)
--- PASS: TestAccAWSGlueCrawler_S3Target (125.84s)
--- PASS: TestAccAWSGlueCrawler_S3Target_SampleSize (128.82s)
--- PASS: TestAccAWSGlueCrawler_JdbcTarget (135.05s)
--- PASS: TestAccAWSGlueCrawler_CatalogTarget (157.83s)
--- PASS: TestAccAWSGlueCrawler_S3Target_Multiple (178.16s)
--- PASS: TestAccAWSGlueCrawler_CatalogTarget_Multiple (256.74s)
--- PASS: TestAccAWSGlueCrawler_DynamodbTarget_scanAll (171.32s)
```

### CODE REVIEW

File: website/docs/r/glue_crawler.html.markdown [hunk 1]
```diff
@@ -168,6 +168,7 @@ The following arguments are supported:
 * `path` - (Required) The path to the Amazon S3 target.
 * `connection_name` - (Optional) The name of a connection which allows crawler to access data in S3 within a VPC.
 * `exclusions` - (Optional) A list of glob patterns used to exclude from the crawl.
+* `sample_size` - (Optional)  Sets the number of files in each leaf folder to be crawled when crawling ample files in a dataset. If not set, all the files are crawled. A valid value is an integer between 1 and 249.
```
Comment (ewbankkit) [inline website/docs/r/glue_crawler.html.markdown:171]: Typo.

```suggestion
* `sample_size` - (Optional)  Sets the number of files in each leaf folder to be crawled when crawling sample files in a dataset. If not set, all the files are crawled. A valid value is an integer between 1 and 249.
```

File: aws/resource_aws_glue_crawler.go [hunk 2]
```diff
@@ -254,15 +249,10 @@ func resourceAwsGlueCrawler() *schema.Resource {
 				},
 			},
 			"recrawl_policy": {
-				Type:     schema.TypeList,
-				Optional: true,
-				MaxItems: 1,
-				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```
Comment (DrFaust92) [inline aws/resource_aws_glue_crawler.go:260]: @ewbankkit do we want a semgrep rule for this?

Comment (ewbankkit) [reply aws/resource_aws_glue_crawler.go:260]: We have #7180 to track use of `suppressMissingOptionalConfigurationBlock`. I'll add a comment about a possible semgrep rule.
Thanks.

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

##### Commercial

```console
% make testacc TESTARGS='-run=TestAccAWSGlueCrawler_'                     
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSGlueCrawler_ -timeout 180m
=== RUN   TestAccAWSGlueCrawler_DynamodbTarget
=== PAUSE TestAccAWSGlueCrawler_DynamodbTarget
=== RUN   TestAccAWSGlueCrawler_DynamodbTarget_scanAll
=== PAUSE TestAccAWSGlueCrawler_DynamodbTarget_scanAll
=== RUN   TestAccAWSGlueCrawler_DynamodbTarget_scanRate
=== PAUSE TestAccAWSGlueCrawler_DynamodbTarget_scanRate
=== RUN   TestAccAWSGlueCrawler_JdbcTarget
=== PAUSE TestAccAWSGlueCrawler_JdbcTarget
=== RUN   TestAccAWSGlueCrawler_JdbcTarget_Exclusions
=== PAUSE TestAccAWSGlueCrawler_JdbcTarget_Exclusions
=== RUN   TestAccAWSGlueCrawler_JdbcTarget_Multiple
=== PAUSE TestAccAWSGlueCrawler_JdbcTarget_Multiple
=== RUN   TestAccAWSGlueCrawler_mongoDBTarget
=== PAUSE TestAccAWSGlueCrawler_mongoDBTarget
=== RUN   TestAccAWSGlueCrawler_mongoDBTarget_scan_all
=== PAUSE TestAccAWSGlueCrawler_mongoDBTarget_scan_all
=== RUN   TestAccAWSGlueCrawler_mongoDBTarget_multiple
=== PAUSE TestAccAWSGlueCrawler_mongoDBTarget_multiple
=== RUN   TestAccAWSGlueCrawler_S3Target
=== PAUSE TestAccAWSGlueCrawler_S3Target
=== RUN   TestAccAWSGlueCrawler_S3Target_ConnectionName
=== PAUSE TestAccAWSGlueCrawler_S3Target_ConnectionName
=== RUN   TestAccAWSGlueCrawler_S3Target_SampleSize
=== PAUSE TestAccAWSGlueCrawler_S3Target_SampleSize
=== RUN   TestAccAWSGlueCrawler_S3Target_Exclusions
=== PAUSE TestAccAWSGlueCrawler_S3Target_Exclusions
=== RUN   TestAccAWSGlueCrawler_S3Target_Multiple
=== PAUSE TestAccAWSGlueCrawler_S3Target_Multiple
=== RUN   TestAccAWSGlueCrawler_CatalogTarget
=== PAUSE TestAccAWSGlueCrawler_CatalogTarget
=== RUN   TestAccAWSGlueCrawler_CatalogTarget_Multiple
=== PAUSE TestAccAWSGlueCrawler_CatalogTarget_Multiple
=== RUN   TestAccAWSGlueCrawler_disappears
=== PAUSE TestAccAWSGlueCrawler_disappears
=== RUN   TestAccAWSGlueCrawler_Classifiers
=== PAUSE TestAccAWSGlueCrawler_Classifiers
=== RUN   TestAccAWSGlueCrawler_Configuration
=== PAUSE TestAccAWSGlueCrawler_Configuration
=== RUN   TestAccAWSGlueCrawler_Description
=== PAUSE TestAccAWSGlueCrawler_Description
=== RUN   TestAccAWSGlueCrawler_Role_ARN_NoPath
=== PAUSE TestAccAWSGlueCrawler_Role_ARN_NoPath
=== RUN   TestAccAWSGlueCrawler_Role_ARN_Path
=== PAUSE TestAccAWSGlueCrawler_Role_ARN_Path
=== RUN   TestAccAWSGlueCrawler_Role_Name_Path
=== PAUSE TestAccAWSGlueCrawler_Role_Name_Path
=== RUN   TestAccAWSGlueCrawler_Schedule
=== PAUSE TestAccAWSGlueCrawler_Schedule
=== RUN   TestAccAWSGlueCrawler_SchemaChangePolicy
=== PAUSE TestAccAWSGlueCrawler_SchemaChangePolicy
=== RUN   TestAccAWSGlueCrawler_TablePrefix
=== PAUSE TestAccAWSGlueCrawler_TablePrefix
=== RUN   TestAccAWSGlueCrawler_RemoveTablePrefix
=== PAUSE TestAccAWSGlueCrawler_RemoveTablePrefix
=== RUN   TestAccAWSGlueCrawler_Tags
=== PAUSE TestAccAWSGlueCrawler_Tags
=== RUN   TestAccAWSGlueCrawler_SecurityConfiguration
=== PAUSE TestAccAWSGlueCrawler_SecurityConfiguration
=== RUN   TestAccAWSGlueCrawler_lineageConfig
=== PAUSE TestAccAWSGlueCrawler_lineageConfig
=== RUN   TestAccAWSGlueCrawler_recrawlPolicy
=== PAUSE TestAccAWSGlueCrawler_recrawlPolicy
=== CONT  TestAccAWSGlueCrawler_DynamodbTarget
=== CONT  TestAccAWSGlueCrawler_disappears
=== CONT  TestAccAWSGlueCrawler_Schedule
=== CONT  TestAccAWSGlueCrawler_recrawlPolicy
=== CONT  TestAccAWSGlueCrawler_Classifiers
=== CONT  TestAccAWSGlueCrawler_SchemaChangePolicy
=== CONT  TestAccAWSGlueCrawler_SecurityConfiguration
=== CONT  TestAccAWSGlueCrawler_JdbcTarget_Exclusions
=== CONT  TestAccAWSGlueCrawler_Role_Name_Path
=== CONT  TestAccAWSGlueCrawler_Role_ARN_Path
=== CONT  TestAccAWSGlueCrawler_Role_ARN_NoPath
=== CONT  TestAccAWSGlueCrawler_Description
=== CONT  TestAccAWSGlueCrawler_Configuration
=== CONT  TestAccAWSGlueCrawler_RemoveTablePrefix
=== CONT  TestAccAWSGlueCrawler_mongoDBTarget_multiple
=== CONT  TestAccAWSGlueCrawler_Tags
=== CONT  TestAccAWSGlueCrawler_CatalogTarget_Multiple
=== CONT  TestAccAWSGlueCrawler_lineageConfig
=== CONT  TestAccAWSGlueCrawler_CatalogTarget
=== CONT  TestAccAWSGlueCrawler_TablePrefix
--- PASS: TestAccAWSGlueCrawler_disappears (61.19s)
=== CONT  TestAccAWSGlueCrawler_S3Target_Multiple
--- PASS: TestAccAWSGlueCrawler_Role_ARN_NoPath (62.56s)
=== CONT  TestAccAWSGlueCrawler_S3Target_ConnectionName
--- PASS: TestAccAWSGlueCrawler_Role_ARN_Path (64.93s)
=== CONT  TestAccAWSGlueCrawler_S3Target_SampleSize
--- PASS: TestAccAWSGlueCrawler_Role_Name_Path (65.13s)
=== CONT  TestAccAWSGlueCrawler_S3Target_Exclusions
--- PASS: TestAccAWSGlueCrawler_JdbcTarget_Exclusions (111.08s)
=== CONT  TestAccAWSGlueCrawler_mongoDBTarget
--- PASS: TestAccAWSGlueCrawler_Description (122.12s)
=== CONT  TestAccAWSGlueCrawler_mongoDBTarget_scan_all
=== CONT  TestAccAWSGlueCrawler_DynamodbTarget_scanRate
--- PASS: TestAccAWSGlueCrawler_RemoveTablePrefix (122.15s)
--- PASS: TestAccAWSGlueCrawler_SecurityConfiguration (122.78s)
=== CONT  TestAccAWSGlueCrawler_JdbcTarget
--- PASS: TestAccAWSGlueCrawler_DynamodbTarget (130.74s)
=== CONT  TestAccAWSGlueCrawler_S3Target
--- PASS: TestAccAWSGlueCrawler_Tags (140.61s)
=== CONT  TestAccAWSGlueCrawler_DynamodbTarget_scanAll
--- PASS: TestAccAWSGlueCrawler_Classifiers (153.25s)
=== CONT  TestAccAWSGlueCrawler_JdbcTarget_Multiple
--- PASS: TestAccAWSGlueCrawler_mongoDBTarget_multiple (162.44s)
--- PASS: TestAccAWSGlueCrawler_Configuration (175.26s)
--- PASS: TestAccAWSGlueCrawler_lineageConfig (186.62s)
--- PASS: TestAccAWSGlueCrawler_SchemaChangePolicy (191.95s)
--- PASS: TestAccAWSGlueCrawler_CatalogTarget_Multiple (202.23s)
--- PASS: TestAccAWSGlueCrawler_recrawlPolicy (202.78s)
--- PASS: TestAccAWSGlueCrawler_Schedule (206.75s)
--- PASS: TestAccAWSGlueCrawler_JdbcTarget (93.06s)
--- PASS: TestAccAWSGlueCrawler_mongoDBTarget (114.31s)
--- PASS: TestAccAWSGlueCrawler_S3Target (107.63s)
--- PASS: TestAccAWSGlueCrawler_DynamodbTarget_scanRate (132.14s)
--- PASS: TestAccAWSGlueCrawler_S3Target_ConnectionName (193.79s)
--- PASS: TestAccAWSGlueCrawler_DynamodbTarget_scanAll (126.23s)
--- PASS: TestAccAWSGlueCrawler_CatalogTarget (275.30s)
--- PASS: TestAccAWSGlueCrawler_TablePrefix (292.59s)
--- PASS: TestAccAWSGlueCrawler_S3Target_SampleSize (304.62s)
--- PASS: TestAccAWSGlueCrawler_mongoDBTarget_scan_all (253.08s)
--- PASS: TestAccAWSGlueCrawler_S3Target_Multiple (341.83s)
--- PASS: TestAccAWSGlueCrawler_S3Target_Exclusions (349.38s)
--- PASS: TestAccAWSGlueCrawler_JdbcTarget_Multiple (525.82s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	682.527s
```

##### GovCloud

```console
% make testacc TESTARGS='-run=TestAccAWSGlueCrawler_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSGlueCrawler_ -timeout 180m
=== RUN   TestAccAWSGlueCrawler_DynamodbTarget
=== PAUSE TestAccAWSGlueCrawler_DynamodbTarget
=== RUN   TestAccAWSGlueCrawler_DynamodbTarget_scanAll
=== PAUSE TestAccAWSGlueCrawler_DynamodbTarget_scanAll
=== RUN   TestAccAWSGlueCrawler_DynamodbTarget_scanRate
=== PAUSE TestAccAWSGlueCrawler_DynamodbTarget_scanRate
=== RUN   TestAccAWSGlueCrawler_JdbcTarget
=== PAUSE TestAccAWSGlueCrawler_JdbcTarget
=== RUN   TestAccAWSGlueCrawler_JdbcTarget_Exclusions
=== PAUSE TestAccAWSGlueCrawler_JdbcTarget_Exclusions
=== RUN   TestAccAWSGlueCrawler_JdbcTarget_Multiple
=== PAUSE TestAccAWSGlueCrawler_JdbcTarget_Multiple
=== RUN   TestAccAWSGlueCrawler_mongoDBTarget
=== PAUSE TestAccAWSGlueCrawler_mongoDBTarget
=== RUN   TestAccAWSGlueCrawler_mongoDBTarget_scan_all
=== PAUSE TestAccAWSGlueCrawler_mongoDBTarget_scan_all
=== RUN   TestAccAWSGlueCrawler_mongoDBTarget_multiple
=== PAUSE TestAccAWSGlueCrawler_mongoDBTarget_multiple
=== RUN   TestAccAWSGlueCrawler_S3Target
=== PAUSE TestAccAWSGlueCrawler_S3Target
=== RUN   TestAccAWSGlueCrawler_S3Target_ConnectionName
=== PAUSE TestAccAWSGlueCrawler_S3Target_ConnectionName
=== RUN   TestAccAWSGlueCrawler_S3Target_SampleSize
=== PAUSE TestAccAWSGlueCrawler_S3Target_SampleSize
=== RUN   TestAccAWSGlueCrawler_S3Target_Exclusions
=== PAUSE TestAccAWSGlueCrawler_S3Target_Exclusions
=== RUN   TestAccAWSGlueCrawler_S3Target_Multiple
=== PAUSE TestAccAWSGlueCrawler_S3Target_Multiple
=== RUN   TestAccAWSGlueCrawler_CatalogTarget
=== PAUSE TestAccAWSGlueCrawler_CatalogTarget
=== RUN   TestAccAWSGlueCrawler_CatalogTarget_Multiple
=== PAUSE TestAccAWSGlueCrawler_CatalogTarget_Multiple
=== RUN   TestAccAWSGlueCrawler_disappears
=== PAUSE TestAccAWSGlueCrawler_disappears
=== RUN   TestAccAWSGlueCrawler_Classifiers
=== PAUSE TestAccAWSGlueCrawler_Classifiers
=== RUN   TestAccAWSGlueCrawler_Configuration
=== PAUSE TestAccAWSGlueCrawler_Configuration
=== RUN   TestAccAWSGlueCrawler_Description
=== PAUSE TestAccAWSGlueCrawler_Description
=== RUN   TestAccAWSGlueCrawler_Role_ARN_NoPath
=== PAUSE TestAccAWSGlueCrawler_Role_ARN_NoPath
=== RUN   TestAccAWSGlueCrawler_Role_ARN_Path
=== PAUSE TestAccAWSGlueCrawler_Role_ARN_Path
=== RUN   TestAccAWSGlueCrawler_Role_Name_Path
=== PAUSE TestAccAWSGlueCrawler_Role_Name_Path
=== RUN   TestAccAWSGlueCrawler_Schedule
=== PAUSE TestAccAWSGlueCrawler_Schedule
=== RUN   TestAccAWSGlueCrawler_SchemaChangePolicy
=== PAUSE TestAccAWSGlueCrawler_SchemaChangePolicy
=== RUN   TestAccAWSGlueCrawler_TablePrefix
=== PAUSE TestAccAWSGlueCrawler_TablePrefix
=== RUN   TestAccAWSGlueCrawler_RemoveTablePrefix
=== PAUSE TestAccAWSGlueCrawler_RemoveTablePrefix
=== RUN   TestAccAWSGlueCrawler_Tags
=== PAUSE TestAccAWSGlueCrawler_Tags
=== RUN   TestAccAWSGlueCrawler_SecurityConfiguration
=== PAUSE TestAccAWSGlueCrawler_SecurityConfiguration
=== RUN   TestAccAWSGlueCrawler_lineageConfig
=== PAUSE TestAccAWSGlueCrawler_lineageConfig
=== RUN   TestAccAWSGlueCrawler_recrawlPolicy
=== PAUSE TestAccAWSGlueCrawler_recrawlPolicy
=== CONT  TestAccAWSGlueCrawler_DynamodbTarget
=== CONT  TestAccAWSGlueCrawler_disappears
=== CONT  TestAccAWSGlueCrawler_mongoDBTarget_multiple
=== CONT  TestAccAWSGlueCrawler_mongoDBTarget_scan_all
=== CONT  TestAccAWSGlueCrawler_recrawlPolicy
=== CONT  TestAccAWSGlueCrawler_lineageConfig
=== CONT  TestAccAWSGlueCrawler_SecurityConfiguration
=== CONT  TestAccAWSGlueCrawler_Tags
=== CONT  TestAccAWSGlueCrawler_RemoveTablePrefix
=== CONT  TestAccAWSGlueCrawler_TablePrefix
=== CONT  TestAccAWSGlueCrawler_SchemaChangePolicy
=== CONT  TestAccAWSGlueCrawler_Schedule
=== CONT  TestAccAWSGlueCrawler_Role_Name_Path
=== CONT  TestAccAWSGlueCrawler_Role_ARN_Path
=== CONT  TestAccAWSGlueCrawler_DynamodbTarget_scanRate
=== CONT  TestAccAWSGlueCrawler_Role_ARN_NoPath
=== CONT  TestAccAWSGlueCrawler_Description
=== CONT  TestAccAWSGlueCrawler_JdbcTarget
=== CONT  TestAccAWSGlueCrawler_JdbcTarget_Exclusions
=== CONT  TestAccAWSGlueCrawler_S3Target_Exclusions
--- PASS: TestAccAWSGlueCrawler_Role_Name_Path (56.77s)
=== CONT  TestAccAWSGlueCrawler_Configuration
--- PASS: TestAccAWSGlueCrawler_Role_ARN_Path (58.99s)
=== CONT  TestAccAWSGlueCrawler_Classifiers
--- PASS: TestAccAWSGlueCrawler_disappears (76.54s)
=== CONT  TestAccAWSGlueCrawler_CatalogTarget
--- PASS: TestAccAWSGlueCrawler_Role_ARN_NoPath (79.74s)
=== CONT  TestAccAWSGlueCrawler_CatalogTarget_Multiple
--- PASS: TestAccAWSGlueCrawler_TablePrefix (122.44s)
=== CONT  TestAccAWSGlueCrawler_DynamodbTarget_scanAll
--- PASS: TestAccAWSGlueCrawler_SchemaChangePolicy (122.85s)
=== CONT  TestAccAWSGlueCrawler_S3Target_Multiple
--- PASS: TestAccAWSGlueCrawler_JdbcTarget (123.12s)
=== CONT  TestAccAWSGlueCrawler_mongoDBTarget
--- PASS: TestAccAWSGlueCrawler_RemoveTablePrefix (127.88s)
=== CONT  TestAccAWSGlueCrawler_JdbcTarget_Multiple
--- PASS: TestAccAWSGlueCrawler_DynamodbTarget (129.71s)
=== CONT  TestAccAWSGlueCrawler_S3Target_ConnectionName
--- PASS: TestAccAWSGlueCrawler_JdbcTarget_Exclusions (130.33s)
=== CONT  TestAccAWSGlueCrawler_S3Target_SampleSize
--- PASS: TestAccAWSGlueCrawler_Description (131.72s)
=== CONT  TestAccAWSGlueCrawler_S3Target
--- PASS: TestAccAWSGlueCrawler_SecurityConfiguration (132.44s)
--- PASS: TestAccAWSGlueCrawler_S3Target_Exclusions (133.18s)
--- PASS: TestAccAWSGlueCrawler_mongoDBTarget_multiple (162.14s)
--- PASS: TestAccAWSGlueCrawler_recrawlPolicy (162.49s)
--- PASS: TestAccAWSGlueCrawler_DynamodbTarget_scanRate (162.95s)
--- PASS: TestAccAWSGlueCrawler_lineageConfig (164.18s)
--- PASS: TestAccAWSGlueCrawler_mongoDBTarget_scan_all (165.31s)
--- PASS: TestAccAWSGlueCrawler_Tags (165.76s)
--- PASS: TestAccAWSGlueCrawler_Schedule (165.86s)
--- PASS: TestAccAWSGlueCrawler_S3Target_ConnectionName (54.26s)
--- PASS: TestAccAWSGlueCrawler_CatalogTarget (111.41s)
--- PASS: TestAccAWSGlueCrawler_Configuration (133.40s)
--- PASS: TestAccAWSGlueCrawler_Classifiers (131.46s)
--- PASS: TestAccAWSGlueCrawler_mongoDBTarget (77.38s)
--- PASS: TestAccAWSGlueCrawler_S3Target (74.34s)
--- PASS: TestAccAWSGlueCrawler_S3Target_SampleSize (76.08s)
--- PASS: TestAccAWSGlueCrawler_DynamodbTarget_scanAll (96.77s)
--- PASS: TestAccAWSGlueCrawler_S3Target_Multiple (99.75s)
--- PASS: TestAccAWSGlueCrawler_CatalogTarget_Multiple (144.02s)
--- PASS: TestAccAWSGlueCrawler_JdbcTarget_Multiple (96.51s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	227.487s
```

**Comment by ewbankkit** [comment]
@DrFaust92 Thanks for the contribution :tada: :clap:.

**Pull request merged**